src/benchmark/tpch_benchmark.cpp

```diff
@@ -49,7 +49,7 @@ int main(int argc, char* argv[]) {
   cli_options.add_options()
```

Comment:
(Random line)

Before I forget: After this PR it will not be possible to run the TPC-H benchmark with the "default"/"Query Validation" parameters, right? I have a slight feeling that having this option available would be desirable for people who'd like to reduce noise in the benchmark?

Reply:
No, and I think that's a good thing. If someone reeeeally wants to do that, they can change it in the query_generator (i.e., use the deterministic queries), but I don't think we should make that easy.

Reply:
okay, will bring this up again as soon as it actually becomes a problem ;)